torchvision/utils.py

```diff
@@ -382,6 +389,132 @@ def draw_keypoints(
     return torch.from_numpy(np.array(img_to_draw)).permute(2, 0, 1).to(dtype=torch.uint8)
 
 
+# Flow visualization code adapted from https://github.com/tomrunia/OpticalFlow_Visualization
+@torch.no_grad()
+def flow_to_image(
+    flow_uv: torch.Tensor,
+    clip_flow: Optional[float] = None,
+) -> torch.Tensor:
+
+    """
+    Converts the two dimensional flow of image to a RGB Image.
+
+    Args:
+        flow_uv (Tensor): Flow UV image of shape [2, H, W]
+        clip_flow (float, optional): Clip maximum of flow values. Defaults to None.
+
+    Returns:
+        img (Tensor[3, H, W]): Image Tensor with flow drawn.
+    """
+
+    if flow_uv.ndim != 3:
+        raise ValueError("Input flow should have 3 dimensions [2, H, W]")
+    if flow_uv.size(0) != 2:
+        raise ValueError("Input flow should have shape [2, H, W]")
+
+    if clip_flow is not None:
+        flow_uv = torch.clip(flow_uv, 0, clip_flow)
+
+    u = flow_uv[0, :, :]
+    v = flow_uv[1, :, :]
+    rad = torch.sqrt(torch.square(u) + torch.square(v))
+    rad_max = torch.max(rad)
+    epslion = 1e-5
+    u = u / (rad_max + epslion)
+    v = v / (rad_max + epslion)
+    return _flow_uv_to_colors(u, v)
+
+
+@torch.no_grad()
+def _flow_uv_to_colors(
+    u: torch.Tensor,
+    v: torch.Tensor,
```

Comment:
instead of acception `u` and `v` separately I think we could just accept a single tensor `normalized_flow`